lib/bridge.js

```diff
@@ -642,13 +651,20 @@ Bridge.prototype.provisionUser = function(matrixUser, provisionedUser) {
         shouldJoinGroup = false;
     }
 
+    var groupVisibility = self.opts.group.memberVisibility;
```

Comment:
This should really only be in the scope of `if (shouldJoinGroup && self.groupId)

Reply:
moved